<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>638</width>
    <height>481</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>638</width>
    <height>481</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>638</width>
    <height>481</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="wallpaper">
    <property name="geometry">
     <rect>
      <x>-20</x>
      <y>-10</y>
      <width>671</width>
      <height>461</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap>wallpaper.png</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="startButton">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>380</y>
      <width>121</width>
      <height>26</height>
     </rect>
    </property>
    <property name="text">
     <string>Start</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="usernameText">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>340</y>
      <width>113</width>
      <height>26</height>
     </rect>
    </property>
    <property name="maxLength">
     <number>32</number>
    </property>
    <property name="placeholderText">
     <string>Username...</string>
    </property>
   </widget>
   <widget class="QComboBox" name="minecraftVersions">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>340</y>
      <width>121</width>
      <height>26</height>
     </rect>
    </property>
    <property name="placeholderText">
     <string>Versions</string>
    </property>
   </widget>
   <widget class="QPushButton" name="saveButton">
    <property name="geometry">
     <rect>
      <x>500</x>
      <y>370</y>
      <width>121</width>
      <height>26</height>
     </rect>
    </property>
    <property name="text">
     <string>Save Settings</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="ramText">
    <property name="geometry">
     <rect>
      <x>500</x>
      <y>330</y>
      <width>113</width>
      <height>26</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="ramLabel">
    <property name="geometry">
     <rect>
      <x>500</x>
      <y>300</y>
      <width>113</width>
      <height>26</height>
     </rect>
    </property>
    <property name="autoFillBackground">
     <bool>true</bool>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-weight:700;&quot;&gt;RAM [MB]:&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>638</width>
     <height>23</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
